### PR TITLE
DEP 4.10 requirements

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripeltd/product-issues/issues/492

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -7,40 +7,40 @@
...
-        "silverstripe/recipe-solr-search": "2.x-dev",
+        "silverstripe/installer": "4.10.x-dev",
+        "silverstripe/recipe-authoring-tools": "1.10.x-dev",
+        "silverstripe/recipe-blog": "1.10.x-dev",
+        "silverstripe/recipe-collaboration": "1.10.x-dev",
+        "silverstripe/recipe-form-building": "1.10.x-dev",
+        "silverstripe/recipe-reporting-tools": "1.10.x-dev",
+        "silverstripe/recipe-services": "1.10.x-dev",
+        "silverstripe/recipe-content-blocks": "2.10.x-dev",
+        "silverstripe/recipe-solr-search": "2.10.x-dev",
```
Comment (maxime-rainville) [inline composer.json:18]: We need to update the matching branch on silverstripe/recipe-solr-search

File: composer.json [hunk 2]
```diff
@@ -7,40 +7,40 @@
...
-        "silverstripe/recipe-ccl": "2.x-dev",
-        "cwp/cwp": "2.x-dev",
-        "cwp/cwp-pdfexport": "1.x-dev",
-        "dnadesign/silverstripe-elemental-userforms": "3.x-dev",
-        "symbiote/silverstripe-multivaluefield": "5.x-dev",
-        "tractorcow/silverstripe-fluent": "4.x-dev"
+        "cwp/starter-theme": "3.2.x-dev",
+        "cwp/agency-extensions": "2.6.x-dev",
+        "cwp/watea-theme": "3.1.x-dev",
+        "silverstripe/recipe-ccl": "2.9.x-dev",
```
Comment (maxime-rainville) [inline composer.json]:
```suggestion
        "silverstripe/recipe-ccl": "2.10.x-dev",
```

File: composer.json [hunk 3]
```diff
@@ -7,40 +7,40 @@
...
+        "silverstripe/recipe-solr-search": "2.10.x-dev",
 
-        "silverstripe/registry": "2.x-dev",
-        "silverstripe/totp-authenticator": "4.x-dev",
-        "silverstripe/mfa": "4.x-dev",
-        "silverstripe/crontask": "2.x-dev",
-        "silverstripe/gridfieldqueuedexport": "2.x-dev",
-        "silverstripe/ldap": "1.x-dev",
+        "silverstripe/registry": "2.4.x-dev",
+        "silverstripe/totp-authenticator": "4.2.x-dev",
```
Comment (maxime-rainville) [inline composer.json]:
```suggestion
        "silverstripe/totp-authenticator": "4.3.x-dev",
```

File: composer.json (hunk 2 above)
Comment (maxime-rainville) [reply composer.json]: Can we move this requirement with tho other recipes?

File: composer.json [hunk 4]
```diff
@@ -7,40 +7,40 @@
...
-        "tractorcow/silverstripe-fluent": "4.x-dev"
+        "cwp/starter-theme": "3.2.x-dev",
+        "cwp/agency-extensions": "2.6.x-dev",
+        "cwp/watea-theme": "3.1.x-dev",
+        "silverstripe/recipe-ccl": "2.9.x-dev",
+        "cwp/cwp": "2.9.x-dev",
+        "cwp/cwp-pdfexport": "1.3.x-dev",
+        "dnadesign/silverstripe-elemental-userforms": "3.1.x-dev",
+        "symbiote/silverstripe-multivaluefield": "5.2.x-dev",
+        "tractorcow/silverstripe-fluent": "4.5.x-dev"
```
Comment (maxime-rainville) [inline composer.json]:
```suggestion
        "tractorcow/silverstripe-fluent": "4.6.x-dev"
```

**Review by maxime-rainville — CHANGES_REQUESTED** [review]
Maybe fix this one once all the other recipes have been fixed

File: composer.json (hunk 1 above)
Comment (emteknetnz) [reply composer.json:18]: 2.10 exists https://github.com/silverstripe/recipe-solr-search/tree/2.10

File: composer.json (hunk 3 above)
Comment (emteknetnz) [reply composer.json]: Updated

File: composer.json (hunk 2 above)
Comment (emteknetnz) [reply composer.json]: Updated

File: composer.json (hunk 4 above)
Comment (emteknetnz) [reply composer.json]: Updated

**Comment by emteknetnz** [comment]
^ broken build requires silverstripe/installer PR to be merged first https://github.com/silverstripe/silverstripe-installer/pull/312

File: composer.json (hunk 1 above)
Comment (maxime-rainville) [reply composer.json:18]: It exists but it needs a PR to update its composer file

Comment (emteknetnz) [reply composer.json:18]: Created https://github.com/silverstripe/recipe-solr-search/pull/3

**Review by maxime-rainville — APPROVED** [review]
LGTM ... we'll deal with any outstanding broken build post merge

**Pull request merged**